crates/control/src/search_suggestor.rs

```diff
@@ -151,3 +161,45 @@ impl IndexMut<Point2<Field>> for Heatmap {
         &mut self.map[heatmap_point]
     }
 }
+
+fn get_rule_hypotheses(
+    primary_state: PrimaryState,
+    filtered_game_controller_state: FilteredGameControllerState,
+    field_dimensions: FieldDimensions,
+) -> Vec<Point2<Field>> {
+    let kicking_team_half =
+        conservative_kicking_team_half(filtered_game_controller_state.kicking_team);
+
+    match (primary_state, filtered_game_controller_state.sub_state) {
+        (PrimaryState::Ready, Some(SubState::PenaltyKick)) => {
+            let kick_half = kicking_team_half.mirror();
+            vec![field_dimensions.penalty_spot(kick_half)]
+        }
+        // Kick-off
+        (PrimaryState::Ready, None) => vec![field_dimensions.center()],
```

Comment:
This both seems like it overlaps/duplicates a bit with the ```rule_ball``` in the ```ball_state_composer```. 
I would suggest maybe using the rule ball as an input for the search suggestor, what do you think?

Reply:
Fair enough